### PR TITLE
Fix first token

### PR DESCRIPTION
**Pull request by sywangyi**
# What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

File: text-generation-inference/server/text_generation_server/utils/logits_process.py [hunk 1]
```diff
@@ -59,16 +58,14 @@ def __call__(self, scores):
...
-        else:
-            # 1st token disposal
-            for warper in self.warpers:
-                scores = warper(None, scores)
-            return scores, torch.log_softmax(scores, -1)
+        return self.static_warped_scores, self.static_next_logprob
+        # cpu branch
+        for warper in self.warpers:
+            scores = warper(None, scores)
+        return scores, torch.log_softmax(scores, -1)
```
Comment (regisss) [inline text-generation-inference/server/text_generation_server/utils/logits_process.py:68]: This part of the code will never be reached as we return before, is it expected?

Comment (sywangyi) [reply text-generation-inference/server/text_generation_server/utils/logits_process.py:68]: yes, the 1st token score also has shape of [1, xxxx], same with the other token. as in https://github.com/huggingface/optimum-habana/pull/408/files#diff-244871853752a7a019853c852803a797a9a14dddb3d9d859ecbae4b316ed6974R658

Comment (regisss) [reply text-generation-inference/server/text_generation_server/utils/logits_process.py:68]: So why keeping these lines then?

Comment (sywangyi) [reply text-generation-inference/server/text_generation_server/utils/logits_process.py:68]: should I remove it. I see https://github.com/huggingface/text-generation-inference/blob/main/server/text_generation_server/utils/logits_process.py#L66 . Is original code left here for debug?

Comment (regisss) [reply text-generation-inference/server/text_generation_server/utils/logits_process.py:68]: I think so given this code will never be reached (the method always returns before reaching there)

**Review by regisss — APPROVED** [review]

**Pull request merged**